tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_test.ts

```diff
@@ -405,5 +446,67 @@ describe('linked_time_fob_controller', () => {
         end: {step: 3},
       });
     });
+
+    it('double clicking fob changes to input', () => {
```

Comment:
These tests operate directly on the linked-time-fob component instead of on the same component as the other tests. It would be good to organize them in a way to reflect that.

Ideally I think they live in their own test file. At the very least they should live in their own top-level describe('linked_time_fob') and not nested in describe('linked_time_fob_controller').